scripts/firedrake-install

```diff
@@ -17,6 +17,7 @@ from itertools import chain
 import re
```

Comment:
You don't appear to have actually add the `--show-dependencies` functionality to this file yet.